src/jc21/CliTable.php

```diff
@@ -400,7 +400,7 @@ protected function getFormattedRow($rowData, $columnLengths, $header = false) {
                 $color = $this->fields[$key]['color'];
             }
 
-            $fieldLength  = strlen($field) + 1;
+            $fieldLength  = mb_strlen($field) + 1;
```

Comment:
If we let the `strlen` replace with `mb_strlen`, we have to define the `ext-mbstring` inside `require` block in `composer.json`.

The sample definition is as follows:

```
......
 "require": {
        "php": ">=5.3.0",
        "ext-mbstring": "*"
},
......
```